db/migrate/20240312122903_make_stripe_payment_intent_generic.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class MakeStripePaymentIntentGeneric < ActiveRecord::Migration[7.1]
+  def change
+    change_table :stripe_payment_intents do |t|
+      t.rename :user_id, :initiated_by_id # Make the purpose of the user_id column more clear
+      t.rename :confirmed_by_id, :confirmation_source_id
+      t.rename :confirmed_by_type, :confirmation_source_type
+      t.rename :canceled_by_id, :cancellation_source_id
+      t.rename :canceled_by_type, :cancellation_source_type
+      t.rename :stripe_record_id, :payment_record_id
+
+      t.string :wca_status
+      t.string :payment_record_type, after: :holder_id
```

Comment:
The `payment_record_type` should definitely go after `payment_record_id`. These two belong together because they identify the polymorphic association `payment_record`.

Reply:
Doesn't the `type` field precede the `id` field on a polymorphic association? I used `competition_payment_integrations` and the `holder` field in `payment_intents` as a reference: 
![image](https://github.com/thewca/worldcubeassociation.org/assets/52967253/ad347f7a-f395-4323-884b-9c35805fbfde)

![image](https://github.com/thewca/worldcubeassociation.org/assets/52967253/ebe27f08-597d-4967-8257-51dae4b271cc)

Reply:
Okay, if that's the way Rails does things then fair enough. Then I assume `holder_id` is indeed the column that happens to be in front of `payment_record_id`?

Reply:
Yep! You can see it in the 2nd screenshot, 3rd from the top